lang/elaborator/src/typechecker/decls/mod.rs

```diff
@@ -85,8 +85,11 @@ fn check_metavars_resolved(meta_vars: &HashMap<MetaVar, MetaVarState>, decls: &[
     for decl in decls {
         if decl.contains_metavars() {
             return Err(TypeError::Impossible {
-                message: format!("Declaration {} contains unresolved metavariables", decl.ident()),
-                span: None,
+                message: format!(
+                    "Declaration {:?} contains unresolved metavariables",
+                    decl.ident()
```

Comment:
This is not a good error message.

Reply:
I think we need to pattern-match here.

Reply:
It is also marked as `Impossible`, so it should not be visible to the user.